docs/DafnyRef/Types.md

```diff
@@ -685,13 +685,16 @@ notation | variance | cardinality-preserving
 - _contra-variance_ (`A<-T>`) means that if `U` is a subtype of `V` then `A<V>` is a subtype of `A<U>`
 - _non-variance_ (`A<T>` or `A<!T>`)  means that if `U` is a different type than `V` then there is no subtyping relationship between `A<U>` and `A<V>`
 
-_Cardinality preserving_ means that the cardinal of the set of all values denoted by the type parameter is not strictly less than the set of all values being defined by the type using this type parameter.
+_Cardinality preserving_ means that the cardinality of the set of all values denoted by the type parameter is not strictly less than that of the set of all values being defined by the type using this type parameter.
```

Comment:
This is okay. However, the phrasing "cardinality [of the] type parameter is not strictly less than [cardinality of] the type" seems to "blame" the type parameter. Would you be alright with switching it around to:

"... means that the cardinality of the type being defined never exceeds the cardinality of any of its type parameters"

Btw, as a fine point, when we speak of cardinality here, we don't intend to distinguish between different _finite_ cardinalities. For example, it's perfectly legal to define `type MyPair<X> = (X, X)`, even though `MyPair<bool>` has greater cardinality than `bool`. I propose that we not mention this fine point.

Reply:
Fixed as you suggest.